Developer tools/Code editors and browsers.md

```diff
@@ -0,0 +1,46 @@
+## Code editors and IDE
+
+We all have our preferred way of writing code. It's something you get accustomed to, and it almost becomes a second-nature to you.
+
+
+Over time you add a few shortcuts here and there. You discover some cool and useful plugins that will enhance your productivity.
+
+
+As time goes by you become well accustomed to your editor and it's a pain to switch to a new cool editor.
+
+And we don't expect you to.
+
+We don't care what you use to write your code. Unless it's on paper. That's a bit hard to maintain, plus it's hard to run automated checks on it 😄.
+
+Today you have a plethora of useful editors and IDEs you can develop your code on.
+
+We'll give you a list of editors/IDEs we are using, or that we used in the past (and they still occupy a special place in our hearts).
+
+### [VS Code](https://code.visualstudio.com/)
+
+A versatile and probably the most used editor in the world at the moment. It's free and has tons of useful plugins you can use. Super useful, especially when writing JS code.
+
+### [PhpStorm](https://www.jetbrains.com/phpstorm/)
+
+Probably the most powerful IDE for writing any kind of PHP code out there. It's a fully fledged IDE that comes with Version Control Systems integration, support for remote deployment, databases/SQL, command-line tools, Docker, Composer, REST Client, and many other tools.
+
+When it comes to any kind of PHP refactoring, PhpStorm is the king.
+
+It is a paid product, that comes with a 30 day free trial.
+
+### [Sublime Text](https://www.sublimetext.com/)
+
+Like VS Code, Sublime is an excellent IDE with plethora of plugins, and add-ons.
+
+ ### [Atom](https://atom.io/)
+
+Before VS Code, there was Atom. It's a versatile editor that you can customize to your liking. It's so popular that it inspired icon sets for both [VS Code](https://marketplace.visualstudio.com/items?itemName=emroussel.atom-icons) and [PhpStorm](https://plugins.jetbrains.com/plugin/10044-atom-material-icons)
+
+ ### Vim, Nano
+If you live in a terminal, the chances are you are using either [Vim](https://www.vim.org/) or [Nano](https://en.wikipedia.org/wiki/GNU_nano). You probably heard of Vim joke about people being [unable to exit it](https://stackoverflow.com/q/11828270/629127). It's not that hard, so you have no reason to fear it 😄
+
+## Browsers
+
+Like with browsers, you can use what your heart desires. Except Internet Explorer. [It's dead and should stay that way](https://docs.microsoft.com/en-us/lifecycle/announcements/internet-explorer-11-end-of-support) 😅.
+
+[Chrome](https://www.google.com/chrome/), [Firefox](https://www.mozilla.org/en-US/firefox/new/), [Safari](https://www.apple.com/safari/), or [Edge](https://www.microsoft.com/en-us/edge), use what you feel the most comfortable with.
```

Comment:
ending of sentence sound a little bit strange, maybe **use whatever you feel most comfortable with**